common/src/test/java/org/apache/celeborn/common/network/RpcIntegrationSuiteJ.java

```diff
@@ -60,6 +63,12 @@ public static void setUp() throws Exception {
         new BaseMessageHandler() {
           @Override
           public void receive(TransportClient client, RequestMessage message) {
+            if (message instanceof RequestMessageV2) {
```

Comment:
do you have plan for ResponseMessage?

Reply:
ResponseMessage is widely used, and its content is too simple to have compatibility issues. So I think there is no need to change it, just putting the needed pb binary array into its managed buffer should be enough.